wremnants/CardTool.py

```diff
@@ -162,6 +166,8 @@ def setNominalName(self, histName):
         if self.datagroups:
             self.datagroups.setNominalName(histName)
 
+    # FIXME ? : this will return True for Fake since it has Data in the list of members.
+    #           Is it intended? I guess isData is also meant to indentify "data-driven" then
     def isData(self, procName):
```

Comment:
I'm not sure which makes sense. If you want it to be all, then change any -> all

Reply:
So right now isMC works by negating isData, so indeed isMC will not include the fakes. It can stay like that probably, or we can define a tighter isData which checks that all members are real data (so this will not include the fakes because it is a mixture, and then isMC could negate the looser one to keep working as now)

Reply:
Added an option to check that all are real data, by default it also accepts data-driven estimates as before, so isMC still works as intended by negating isData and thus accepting only purely MC-based processes